start-cb.sh

```diff
@@ -221,6 +222,7 @@ start_uaa() {
 
     debug "waits for uaadb get registered in consul"
     wait_for_service uaadb
+    sleep 20
```

Comment:
why is the `sleep` coming back again? The `wait_for_service` was created to replace random sleeps